filament/src/driver/ExternalContext.cpp

```diff
@@ -56,7 +58,7 @@ ExternalContext* ExternalContext::create(Backend* backend) noexcept {
         *backend = Backend::OPENGL;
     }
     if (*backend == Backend::VULKAN) {
-        #if defined(FILAMENT_DRIVER_SUPPORTS_VULKAN)
+        #if defined(FILAMENT_DRIVER_SUPPORTS_VULKAN) && defined(USE_EXTERNAL_GLES3)
```

Comment:
Why this test? This is only when the backend is set to Vulkan.

Reply:
will remove